### ISSUE TITLE
We don't set the terminal difficulty in the EL genesis data even though the struct expects it

### ISSUE THREAD

**Issue by h4ck3rk3y** [kurtosis-tech/eth2-merge-kurtosis-module#147]
https://github.com/kurtosis-tech/eth2-merge-kurtosis-module/blob/c7e67c5646eb18a94b1b9ab11c478c7c58002a60/kurtosis-module/impl/participant_network/prelaunch_data_generator/el_genesis/el_genesis_data_generator.go#L89

**Comment by gbouv** [comment]
Yup good point, those are irrelevant now that we're not doing the merge anymore. Submitted a PR to clean those up.
The ETH folks merged a PR on the data generator, we could use it in a more efficient way right now I believe (I think we could completely remove the templating logic). I looked into it on Friday but adapting the module is not trivial, so I figured it should be properly prioritised before I burn too much time. I'll file another issue to track

**Comment by gbouv** [comment]
Files kurtosis-tech/eth2-package#56
